src/FX.GrpcClient/FX.GrpcClient.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);CS8981</WarningsNotAsErrors>
```

Comment:
@webwarrior-ws please add a comment here explaining why this warning needs to be ignored

Reply:
Added comments.